src/tools/cephfs/JournalTool.cc

```diff
@@ -85,21 +92,24 @@ int JournalTool::main(std::vector<const char*> &argv)
     return -EINVAL;
   }
 
+  dout(10) << "JournalTool::main " << dendl;
   std::vector<const char*>::iterator arg = argv.begin();
 
+  dout(10) << "JournalTool::main " << dendl;
   std::string rank_str;
   if(!ceph_argparse_witharg(argv, arg, &rank_str, "--rank", (char*)NULL)) {
     // Default: act on rank 0.  Will give the user an error if they
     // try invoking this way when they have more than one filesystem.
     rank_str = "0";
   }
 
+  dout(10) << "JournalTool::main rank_str=" << rank_str << dendl;
```

Comment:
Again -- this looks unrelated. I'm thinking you copied code from the docker container and then issued the pull request. So its confusing as to what your code is versus mine and the Ceph guy's.